python_modules/dagster/dagster/_core/storage/event_log/schema.py

```diff
@@ -75,3 +91,6 @@
     SqlEventLogStorageTable.c.id,
     mysql_length={"dagster_event_type": 64},
 )
+db.Index(
+    "idx_asset_event_tags", AssetEventTagsTable.c.key, AssetEventTagsTable.c.value, mysql_length=64
+)
```

Comment:
confident this is what we need? i dont have context on expected access patterns

Reply:
is `64` the right length? Im not sure how those work or what restrictions we put on the contents of keys/values

Reply:
I think in the primary views that we'd like to support are all in terms of a specific asset, i.e. showing how many partitions exist/are missing, so I think I should change this to index by asset key.

As of now, the tags are system populated. They'll be of the format `dagster/partition/dimension_name`. I'm not really sure how these index lengths work, but as as long as the user-specified dimension name doesn't exceed 46 characters we should be able to index properly